daemon/src/model/cfd.rs

```diff
@@ -976,10 +1000,15 @@ impl Cfd {
             ));
         }
 
+        let commit_tx = match dlc.signed_commit_tx() {
+            Ok(commit_tx) => commit_tx,
+            Err(err) => return Err(DecryptCetError::ConstructingSignedCetFailed(err)),
```

Comment:
I think this is using the wrong error variant. Also, you can replace this with a `map_err` :)